### ISSUE TITLE
Pages should contain a unique <title> within <head>

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital.gov.bc.ca#239]
Reference: [WCAG 2.4.2](https://www.w3.org/TR/WCAG21/#page-titled)

### The Problem

If you inspect the `<head>` section of our pages, you'll notice that they all contain `<title>digital.gov.bc.ca</title>`. 

According to [WCAG 2.4.2](https://www.w3.org/TR/WCAG21/#page-titled), this shouldn't be the case. Each page's `<head>` should contain a unique `<title>`, specific to that page. (If you inspect on the [original site](https://digital.gov.bc.ca), you'll notice the titles are all unique.)

### For Discussion
I'm not sure whether the titles should simply and strictly reflect all of the content hierarchy (i.e. mirror the breadcrumb), or if each one should be composed by a human using their best judgment. For example, it might be useful to title the homepage "Digital Government - Province of British Columbia", but as we go further into the site structure, be able to title the **Digital Principles** page "Digital Principles for the Government of British Columbia" rather than have it come out as "Digital Principles for the Government of British Columbia 1.0 - Resources - Digital Government - Province of British Columbia" (way too long). 

### Acceptance Criteria
Modify the code so that the `<head>` of each page contains a unique `<title>` that reflects the title/subject of the page.

### Definition of Done

- [ ] All tests (unit/functional/security/code cov.) pass
- [ ] Code review has been done
- [ ] Pipeline builds
- [ ] PO signs off on the build

**Comment by HeatherRemacle** [comment]
Not sure if this is helpful:

https://www.w3schools.com/tags/tag_title.asp  --> https://www.w3schools.com/jsref/dom_obj_title.asp
